src/App.js

```diff
@@ -0,0 +1,102 @@
+import React, {Component} from 'react';
+import FilmsList from './search/FilmsList';
+import Filter from './common/Filter';
+import Search from "./search/Search";
```

Comment:
Please try to use only one kind of quote. So, I'd recommend you using prettier and/or eslint to automate that